static/js/settings_users.js

```diff
@@ -390,7 +390,7 @@ function start_data_load() {
     populate_users();
 }
 
-function get_human_profile_data(fields_user_pills) {
+export function get_human_profile_data(fields_user_pills) {
```

Comment:
We should probably plan to move this to another file; possible a `custom_profile_fields_ui.js`. (Also noting the TODO comment just below, that will likely be a step in the direction of deduplicating this with `settings_account`.)

It'd be great to do this code move as an independent preparatory commit that just does that one thing (plus of course changes to imports/tests to have it work).